server/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
+  },
+  "author": "",
+  "license": "ISC"
+}
```

Comment:
how come this is being committed? i thought we only needed it in the root directory?

Reply:
I don't really know, i'm not very familiar with what that is

Reply:
@ZahraaTayyar  it is there because you installed npm on your server.